components/raftstore/src/store/config.rs

```diff
@@ -621,6 +626,16 @@ impl std::ops::Deref for RaftstoreConfigManager {
     }
 }
 
+#[derive(Copy, Clone, Debug, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "kebab-case")]
+pub enum ConsistencyCheckMethod {
+    /// Do consistency check for regions based on MVCC.
+    Mvcc,
```

Comment:
This should be put into coprocessor, eg. add a consistency check observer.

Reply:
Do you think we can also move raw consistency check into observer or not?

Reply:
Yes. I think it should work similar as split checker.